### ISSUE TITLE
Runtime issue: Cannot read property 'data' of undefined

### ISSUE THREAD

**Issue by kyekhilevsky** [dequelabs/axe-core#2979]
<< Please describe how you encountered this issue >>

axe-core version: `4.2.0`
axe-extension version: `4.9.3`

```
TypeError: Cannot read property 'data' of undefined
    at Xl._l [as after] (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:289593)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:324563
    at Array.forEach (<anonymous>)
    at ec.after (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:324396)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:332384
    at Array.map (<anonymous>)
    at e.value (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:332240)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:341091
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:97118
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:100990
```

**Comment by straker** [comment]
Thanks for the issue. This is a duplicate of #2944 and #2928. We just released version 4.2.2 which should fix the problem. The extensions should be updated soon as well.

**Comment by kyekhilevsky** [comment]
I had my trial Pro version and it is expired. How can I get the update for the new released version?

From: Steven Lambert ***@***.***>
Reply-To: dequelabs/axe-core ***@***.***>
Date: Thursday, June 3, 2021 at 2:52 PM
To: dequelabs/axe-core ***@***.***>
Cc: Katya Yekhilevsky ***@***.***>, Author ***@***.***>
Subject: Re: [dequelabs/axe-core] Runtime issue: Cannot read property 'data' of undefined (#2979)

[External Email]


Thanks for the issue. This is a duplicate of #2944<https://urldefense.com/v3/__https:/github.com/dequelabs/axe-core/issues/2944__;!!N4vogdjhuJM!U-5gQHnCCVdjZI1v3V6DMyrqjWILavay5wbANUlENMBuCgz_1itiz8oH2ommE5b0YTeoqrzzDug$> and #2928<https://urldefense.com/v3/__https:/github.com/dequelabs/axe-core/issues/2928__;!!N4vogdjhuJM!U-5gQHnCCVdjZI1v3V6DMyrqjWILavay5wbANUlENMBuCgz_1itiz8oH2ommE5b0YTeoGuwAuPI$>. We just released version 4.2.2 which should fix the problem. The extensions should be updated soon as well.

**Comment by dylanb** [comment]
@kyekhilevsky when we update the extension to leverage the new axe-core version, your extension should update automatically. This will probably happen next week
